### ISSUE TITLE
Storage Access API

### ISSUE THREAD

**Issue by Bettelstab** [Fyrd/caniuse#4812]
https://developer.mozilla.org/en-US/docs/Web/API/Storage_Access_API

**Comment by Schweinepriester** [comment]
The MDN link lists https://caniuse.com/#feat=mdn-api_document_requeststorageaccess at the bottom.

Coming to Edge and Chrome it seems: https://twitter.com/tomayac/status/1281149537768345600

**Comment by Schweinepriester** [comment]
Updated in Safari 14.1: <https://webkit.org/blog/11648/new-webkit-features-in-safari-14-1/>

**Comment by GPHemsley** [comment]
Indeed, while the term "Storage Access API" is not used explicitly, `Document.hasStorageAccess()`, `Document.requestStorageAccess()`, and `<iframe sandbox="allow-storage-access-by-user-activation">` are covered by MDN data:
https://caniuse.com/mdn-api_document_hasstorageaccess
https://caniuse.com/mdn-api_document_requeststorageaccess
https://caniuse.com/mdn-html_elements_iframe_sandbox-allow-storage-access-by-user-activation

(via mdn/browser-compat-data#2990)
